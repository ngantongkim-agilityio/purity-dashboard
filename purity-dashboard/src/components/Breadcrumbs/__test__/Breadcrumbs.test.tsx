import { render, fireEvent } from '@testing-library/react';

// Components
import { Breadcrumbs } from '..';

describe('Breadcrumbs component', () => {
  it('Should render Breadcrumbs component with default props', () => {
    const { container } = render(<Breadcrumbs breadcrumbs={[]} />);

    expect(container).toMatchSnapshot();
  });
});
