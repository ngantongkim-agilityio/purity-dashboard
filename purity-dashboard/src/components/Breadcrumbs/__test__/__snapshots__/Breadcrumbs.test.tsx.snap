// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumbs component Should render Breadcrumbs component with default props 1`] = `
<div>
  <nav
    aria-label="Breadcrumb"
    class="mb-6 block"
  >
    <ol
      class="flex text-xl md:text-2xl"
    />
  </nav>
</div>
`;
